docs/config.md

```diff
@@ -237,7 +237,25 @@ templates.
   * `templates.log_node` for commits (with `Option<Commit>` keywords)
   * `templates.op_log_node` for operations (with `Operation` keywords)
 
-For example:
+By default, the following graph nodes are shown to the user:
+
+  * The working-copy-commit is the `@` symbol, and is green.
+  * Immutable nodes (as specified by the `immutable_heads()` revset) are
+    represented with blue diamonds `◆`
+  * Mutable commits are represented with ordinary uncolored circles `○`
+  * "Elided nodes" are specified with the "harpoon" symbol `⇋` to show that
+    there are commits between the two nodes that are "cut" out, and not shown.
+  * Commits whose description begins with `wip:` are represented with a yellow
+    `!` symbol.
```

Comment:
Is this still true?

We also need to remember to update this if we change any of the notes.